controls/combobox/header-footer.md

```diff
@@ -1,19 +1,19 @@
 ---
 title: Header and Footer
 page_title: .NET MAUI ComboBox Documentation - Templates
-description: Check our &quot;Header and Footer&quot; documentation article for Telerik ComboBox for .NET MAUI control.
+description: Learn more about the header and footer templates to the dropdown list of Telerik UI For .NET MAUI ComboBox control.
 position: 9
 slug: combobox-header-footer
 ---
 
 # Header and Footer in .NET MAUI ComboBox
 
-You can easily add Header and Footer to the DropDown list of the ComboBox control through the following properties:
+You can add Header and Footer to the DropDown list of the ComboBox control through the following properties:
 
 * `HeaderTemplate`(`DataTemplate`)&mdash;Defines the template of the header that will be visualized in the drop down list.
 * `FooterTemplate`(`DataTemplate`)&mdash;Defines the template of the footer that will be visualized in the drop down list.
 
-> It is not recommended to add controls in the Header and Footer which steal the focus (like Entry, editor, etc.) from the ComboBox control, as unexpected behavior may occur. 
+> It's not recommended to add controls in the Header and Footer which steal the focus (like Entry, editor, etc.) from the ComboBox control, as unexpected behavior may occur. 
```

Comment:
Replacing "it is" with "it's" doesn't improve the readability,
Find a verb + noun pair that carries the actual meaning. Another option is to rephrase the sentence.